setup.sh

```
@@ -0,0 +1,53 @@
+
+# Upgrade pip
+echo -e "${GREEN}⬆️  Upgrading pip...${NC}"
+pip install --upgrade pip
+
+# Install dependencies
+echo -e "${GREEN}📦 Installing dependencies...${NC}"
+pip install -e .
+pip install -r requirements-dev.txt
+
+# Set up pre-commit hooks
+echo -e "${GREEN}🔧 Setting up pre-commit hooks...${NC}
```

Shellcheck (0.10.0)

[warning] Did you forget to close this double quoted string?

(SC1078)